modules/cnf-fast-event-notifications-api-reference-v2.adoc

```diff
@@ -489,14 +489,14 @@ Returns the current state of the `os-clock-sync-state`, `clock-class`, `lock-sta
     "values": [
       {
         "resource": "/cluster/node/compute-1.example.com/ens2fx/master",
-        "dataType": "notification",
-        "valueType": "enumeration",
+        "data_type": "notification",
+        "value_type": "enumeration",
         "value": "SYNCHRONIZED"
       },
       {
         "resource": "/cluster/node/compute-1.example.com/ens2fx/master",
```

Comment:
`resource`  should change to `ResourceAddress`

Reply:
Thanks @jzding I've corrected both instances